bundles/org.openhab.core.io.rest.ui/src/main/java/org/openhab/core/io/rest/ui/internal/UIResource.java

```diff
@@ -125,10 +127,9 @@ public Response getComponentByUID(@PathParam("namespace") String namespace,
     @RolesAllowed({ Role.ADMIN })
     @Path("/components/{namespace}")
     @Produces({ MediaType.APPLICATION_JSON })
-    @ApiOperation(value = "Add an UI component in the specified namespace.", authorizations = {
-            @Authorization(value = "oauth2", scopes = {
-                    @AuthorizationScope(scope = "admin", description = "Admin operations") }) })
-    @ApiResponses(value = { @ApiResponse(code = 200, message = "OK", response = Tile.class) })
```

Comment:
Thanks for fixing the response class 😊 I've been meaning to take care of it, but 🤷

Reply:
These copy/paste errors and wrong responses probably only become a major issue when using the spec with codegen for generating your own model classes. I think most users wouldn't really notice if they only use the Swagger UI for exploring the API. :-)